### PR TITLE
ECVOPRF->VOPRF

### PR DESCRIPTION
**Pull request by alxdavids**
Related to #13

### CODE REVIEW

File: draft-sullivan-cfrg-ecvoprf.md [hunk 1]
```diff
@@ -677,7 +718,7 @@ Y: 04006b0413e2686c4bb62340706de7723471080093422f02dd125c3e72f3507b9200d11481468
 
 # Applications
 
-This section describes various application of ECVOPRF.
+This section describes various applications of our VOPRF protocol.
```
Comment (chris-wood) [inline draft-sullivan-cfrg-ecvoprf.md]: Maybe we should drop "our" here?

Comment (alxdavids) [reply draft-sullivan-cfrg-ecvoprf.md]: Fixed

Comment (alxdavids) [reply draft-sullivan-cfrg-ecvoprf.md]: Fixed

**Comment by alxdavids** [comment]
@chris-wood Sure, I am happy for you to submit. I've made the change to remove "our" and now merging

**Comment by alxdavids** [comment]
@chris-wood Sure, I am happy for you to submit this. I fixed the suggestion you made above, and now going to merge.

**Comment by alxdavids** [comment]
@chris-wood Sure, I am happy for you to submit this. I fixed the suggestion you made above, and now going to merge (if github actually allows me...).åç

**Comment by alxdavids** [comment]
@chris-wood Sure, I am happy for you to submit this. I fixed the suggestion you made above, and now going to merge (if github actually allows me...).

**Pull request merged**